pkg/nsx/services/nsxserviceaccount/cluster.go

```diff
@@ -181,6 +181,7 @@ func (s *NSXServiceAccountService) CreateOrUpdateNSXServiceAccount(ctx context.C
 	// update NSXServiceAccountStatus
 	obj.Status.Phase = v1alpha1.NSXServiceAccountPhaseRealized
 	obj.Status.Reason = "Success."
```

Comment:
I think Reason and Condition.Reason should be single camel case word without "." period.

If the Antrea-NSX supervisor registration code doesn't rely on this period, let's remove the period.

Reply:
OK, let's use Message instead of Reason.

Reply:
it looks like you did not change it yet

Reply:
This part is already merged. Do we need to change it since it is deprecated now.

Reply:
Yes. Let's remove the ending period from "Success." if Antrea-NSX registration code doesn't rely on this period.

Reply:
Removed the period.